docs/book/start/development.md

```diff
@@ -35,7 +35,7 @@ cargo install cargo-generate
 ```
 
 And finally to generate bindings for kernel data structures, you must install
-`bpftool`, either from your distribution or building it from
+`bpftool`. This tool is part of the package `linux-tools-common`, `linux-lowlatency-tools-common` or you can build it from
```

Comment:
These names are unlikely to be the same across distros? Also IIRC linux-tools-common under ubuntu installs a stub that when executed tells you "you need to install linux-tools-<version>" or something like that

Reply:
You're right but it was not clear for me,btw I rewrite it